src/explore-education-statistics-admin/src/components/EditableAccordionSection.tsx

```diff
@@ -31,9 +34,12 @@ const EditableAccordionSection = ({
   onToggle,
   onHeadingChange,
   canEditHeading,
+  onRemoveSection,
+  canRemoveSection = !!onRemoveSection,
```

Comment:
No need to have this boolean check, just check on `onRemoveSection` being a function